tools/data_processing/aia_download_1d.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python
+ 
```

Comment:
Lot's of commented code here and it isn't always clear what is happening. The download scripts are very important to have in good shape...

I'll need to run these and get everything in good order for the submission.